BookHub/userProfile/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 
-# Create your models here.
+
+class Profile(models.Model):
+    user = models.OneToOneField(User,related_name='profile',on_delete=models.CASCADE)
+    profile_pic = models.ImageField(upload_to='images/profile_pics',blank=True)
+    bio = models.TextField(blank=True)
+    favorite_genres = models.CharField(max_length=255, blank=True)
+
+    def __str__(self):
+        return self.user.username
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user=instance)
+
+@receiver(post_save, sender=User)
```

Comment:
```suggestion
@receiver(post_save, sender=User)
```

remove this part as it's not needed